instrumenter_test.go

```diff
@@ -90,6 +90,114 @@ func Test_instrumenter(t *testing.T) {
 				map[ast.Stmt]ast.Stmt{},
 				false,
 				nil,
+				false,
+			}
+			i.instrumentFileNode(f)
+
+			var sb strings.Builder
+			err = printer.Fprint(&sb, fset, f)
+			if err != nil {
+				t.Fatal(err)
+			}
+			if len(i.conds) > 0 {
+				sb.WriteString("\n")
+			}
+			for _, cond := range i.conds {
+				location := strings.TrimPrefix(cond.pos, "test.go")
+				sb.WriteString(fmt.Sprintf("// %s: %q\n",
+					location, cond.text))
+			}
+			actual := sb.String()
+
+			if actual != expected {
+				err := os.WriteFile(base+".gobco", []byte(actual), 0o666)
+				if err != nil {
+					t.Fatal(err)
+				}
+				t.Errorf("expected:\n%s\nactual:\n%s\n", expected, actual)
+			}
+		})
+	}
+}
+
+func Test_instrumenter_c1(t *testing.T) {
```

Comment:
Instead of duplicating this whole code, what about duplicating only the expected output files: one named `.gobco` for branch coverage and one named `.gococo` for condition coverage? That would reduce the number of files and place the two output files for the different modes next to each other, allowing to easily navigate between them and compare them.

When I had to read through this test code (that I wrote myself a year ago), there was a lot of fine-grained detail code that didn't allow me to quickly get an overview. To run the instrumenter in both modes, most of the test code would have to be extracted to an inner function and then be called twice: once for branch coverage mode, once for condition coverage mode.